### PR TITLE
Fix(web): Trick to start animation when inject backdrop (refs #DS-218)

### PR DESCRIPTION
**Pull request by literat**
Using this quick fix https://www.charistheo.io/blog/2021/02/restart-a-css-animation-with-javascript/#restarting-a-css-animation to restart animation when backdrop is injected.

### CODE REVIEW

File: packages/web/src/js/utils/index.ts [hunk 1]
```diff
@@ -28,4 +28,12 @@ const getElementFromSelector = (element: HTMLElement): HTMLElement | null => {
   return selector ? document.querySelector(selector) : null;
 };
 
-export { isElement, getElement, getSelector, getElementFromSelector };
+/**
+ * Trick to restart an element's animation
+ * @see https://www.charistheo.io/blog/2021/02/restart-a-css-animation-with-javascript/#restarting-a-css-animation
+ */
+const reflow = (element: HTMLElement): void => {
+  element.offsetHeight;
```
Comment (adamkudrna) [inline packages/web/src/js/utils/index.ts:36]:
```suggestion
  element.offsetHeight();
```

?

File: packages/web/src/js/Modal.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import SelectorEngine from './dom/SelectorEngine';
 import EventHandler from './dom/EventHandler';
 import BaseComponent from './BaseComponent';
-import { getElementFromSelector } from './utils/index';
+import { getElementFromSelector, reflow } from './utils/index';
```
Comment (adamkudrna) [inline packages/web/src/js/Modal.ts:4]: Is it necessary to specify `/index` in the path?

**Review by adamkudrna — APPROVED** [review]
A small thing in the commit message:

> Trick to start animation when [ _injecting_ backdrop | _backdrop is injected_ ]

or even:

> Force animation when … ⬆️

File: packages/web/src/js/Modal.ts (hunk 2 above)
Comment (literat) [reply packages/web/src/js/Modal.ts]: It is not, fixed.

**Pull request merged**